sdk/keyvault/keyvault-keys/README.md

```diff
@@ -226,11 +226,12 @@ await client.updateKey(keyName, result.properties.version, {
 
 ### Deleting a key
 
-The `deleteKey` method sets a key up for deletion. This process will
+The `beginDeleteKey` method sets a key up for deletion. This process will
```

Comment:
Maybe this should mention that you're starting the deletion? Sets a key up makes it seem like this isn't also initiating the delete (perhaps it doesn't in which case this sentence is 100% correct!)